src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Players/PlayersListViewModel.cs

```diff
@@ -30,14 +29,14 @@ public PlayersListViewModel(IQueryable<Player> source, int? index, int size, str
 
             if ((PageNumber > NumberOfPages) || (PageNumber < FIRST_PAGE))
             {
-                throw new ArgumentOutOfRangeException();
+                throw new ArgumentOutOfRangeException(nameof(index));
             }
-
+#pragma warning disable S2971 // Query does not work without ToList() in the middle of the chain
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)

Reply:
Why it does not work?